src/helpers/helpers.sh

```diff
@@ -43,6 +43,7 @@ if [ "$(is_git_repository)" == true ]; then
   export INITIAL_BRANCH_NAME=$(get_current_branch_name)
   export INITIAL_DIRECTORY=$(pwd)
   export INITIAL_OPEN_CHANGES=$(has_open_changes)
+  export INITIAL_PREVIOUS_BRANCH_NAME=$(get_previous_branch_name)
```

Comment:
I think it's much more clear knowing that it's the initial state, and there's parity with `INITIAL_BRANCH_NAME`